pkg/deploy/spindeploy/save.go

```diff
@@ -112,38 +113,53 @@ func (d *Deployer) patch(original runtime.Object) error {
 	}
 
 	rsc, _ := apimeta.UnsafeGuessKindToResource(gvk)
-	// gvk.GroupKind().Group
-	// e := d.rawClient.CoreV1().Services(o.GetNamespace())
-	// o.GetResourceVersion()
-	// _, rsc := gvk.GetResourceVersion()
-	cp := original.DeepCopyObject()
+	originalRaw := modifiedRaw.DeepCopyObject()
 
-	err = i.Get().
-		Namespace(o.GetNamespace()).
-		Resource(rsc.Resource).
-		Name(o.GetName()).
-		Do().
-		Into(cp)
+	// avoid reading from cache
```

Comment:
Doesn't it read from the `client` cache on the contrary?

Reply:
I'm not sure which client uses cache or not, I originally tested with the `RESTClient` produced by `Clientset` and it was reading a stale version of the resource that didn't have the latest changes (producing again an infinite loop). 
Then I changed to this other client in `Deployer` struct and it's reading the correct information every time.